scrapers.js

```diff
@@ -1686,6 +1686,41 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'FRA',
+    url: 'https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.csv',
```

Comment:
Make this scraper the authority:
```suggestion
    url: 'https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.csv',
    priority: 1,
```